### ISSUE TITLE
Informed Consent should not decide what happens when we click cancel

### ISSUE THREAD

**Issue by LarsRefsgaard** [carp-dk/research.package#82]
in /src/ui/RPUITask.dart line 202-204
`Navigator.of(context).pop(), // Dismissing the pop-up`
needs to be revised to be customisable or something, because in the Study app we are not pushing the UI on top, but rather moving to that part of the onboarding process UI.

**Comment by bardram** [comment]
I'm not sure this is correct. Everything in RP builds upon the assumption that you push any RPUITask on top of the app.

**Comment by LarsRefsgaard** [comment]
Yes but it also assumes that the task UI is pushed, which might not be correct.
